domains/saikesarla.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "saikesarla",
+           "email": "saikesarla.projects@gmail.com",
+           "discord": "822871034377011200"
+        },
+    
+        "record": {
+            "CNAME": "github.com/saikesarla/portfolio-website/blob/main/index.html"
```

Comment:
You need to add your Github Pages CNAME record, please go here if you don't know what to do https://www.is-a.dev/docs/github-pages/